### PR TITLE
update to rrn62, fix errors

### PR DESCRIPTION
**Pull request by idkjs**
This PR updates the reason-react, bs-platform, and reason-react-native versions. A few changes were required to get the project to build which can be easily seen in the diff.

Thanks.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,12 +35,12 @@
     "test": "yarn re:clean-build"
   },
   "devDependencies": {
-    "bs-platform": "^5.2.0",
+    "bs-platform": "^8.0.3",
```
Comment (MoOx) [inline package.json]:
```suggestion
    "bs-platform": "^8.0.0",
```

File: package.json [hunk 2]
```diff
@@ -35,12 +35,12 @@
...
     "husky": "^1.3.0",
     "lint-staged": "^10.1.1",
     "prettier": "^1.18.0",
-    "reason-react": "^0.7.0",
-    "reason-react-native": "^0.60.0"
+    "reason-react": "^0.9.1",
```
Comment (MoOx) [inline package.json]:
```suggestion
    "reason-react": "^0.9.0",
```

**Review by MoOx — APPROVED** [review]
I like to keep version without patch, to not have too many diff between all repos, so I will commit that.
Thanks a lot for the changes!

**Comment by idkjs** [comment]
What do you mean by versions with out patch, brother? I just checked the package.json. The last number in the version name is the patch. Got it. I will try to remember for next time on this repo.

Peace to you, brother.

**Pull request merged**